### PR TITLE
feat(PE-6553): Evolve ANTs workflow

### PR DESCRIPTION
**Pull request by atticusofsparta**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,12 +21,11 @@
   },
   "dependencies": {
     "@ant-design/icons": "5.3.2",
-    "@ar.io/sdk": "2.1.0",
+    "@ar.io/sdk": "2.1.0-alpha.11",
```
Comment (kunstmusik) [inline package.json:24]: Looks like this went backwards in version?

File: package.json [hunk 2]
```diff
@@ -65,6 +65,8 @@
     "@typescript-eslint/eslint-plugin": "^5.42.1",
     "@typescript-eslint/parser": "^6.19.0",
     "@vitejs/plugin-react": "^4.0.0",
+    "arconnect": "^1.0.3",
```
Comment (kunstmusik) [inline package.json:68]: Shouldn't this be a regular dependency vs. dev dependency?

File: .yarnrc.yml
Comment (dtfiedler) [inline .yarnrc.yml:1]: intent here? was this intentional? we've used yarn berry before but moved away from it as it's overhead wasn't giving us much

**Review by dtfiedler — CHANGES_REQUESTED** [review]
This PR is out of date and has some un-adressed comments

File: package.json (hunk 2 above)
Comment (atticusofsparta) [reply package.json:68]: the name is misleading, its a types package

File: .yarnrc.yml
Comment (atticusofsparta) [reply .yarnrc.yml:1]: I was running into issues with the tailwind install I think relating to yarn, ill revert and see what the issue was here again

Comment (atticusofsparta) [reply .yarnrc.yml:1]: It was a build issue with ant-design dep issues that I was able to fix without bumping yarn, so reverted to 1.22

File: src/components/inputs/buttons/CopyTextButton/CopyTextButton.tsx [hunk 3]
```diff
@@ -71,18 +71,21 @@ function CopyTextButton({
             padding: '4px 8px',
           }}
         >
-          <CopyIcon
-            className="flex"
-            height={size ?? '15px'}
-            width={size ?? '15px'}
-            fill={'inherit'}
+          <div
+            className="flex tems-center justify-center"
```
Comment (kunstmusik) [inline src/components/inputs/buttons/CopyTextButton/CopyTextButton.tsx:75]: typo: items-center

File: package.json [hunk 4]
```diff
@@ -107,5 +115,6 @@
     "**/*.{ts,tsx,js,md,json}": [
       "prettier --write ."
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22"
```
Comment (dtfiedler) [inline package.json:119]: any particular reason for introducing this? if so can we separate it into a specific PR. Jamming these things in with feature changes makes creates noise

File: src/components/layout/ExtendLease/ExtendLease.tsx [hunk 5]
```diff
@@ -159,7 +159,7 @@ function ExtendLease() {
           className="flex flex-row center"
           style={{ justifyContent: 'space-between' }}
         >
-          <h1 className="white">Extend Lease</h1>
+          <h1 className="white text-2xl">Extend Lease</h1>
```
Comment (kunstmusik) [inline src/components/layout/ExtendLease/ExtendLease.tsx:162]: Not sure it's intentional, but this is smaller than what was previously in prod. (Renders as 24px size here). It's odd as 2xl should be 32px with a 16px base font size; unsure if the h1 is interfering somehow.

Comment (kunstmusik) [reply src/components/layout/ExtendLease/ExtendLease.tsx:162]: Ah, it is 24px for 2xl. To get 32px use text-[2rem]

File: src/components/layout/UpgradeUndernames/UpgradeUndernames.tsx [hunk 6]
```diff
@@ -113,7 +113,7 @@ function UpgradeUndernames() {
   return (
     <div className="page center">
       <div className="flex flex-column" style={{ maxWidth: '1000px' }}>
-        <h1 className="flex white">Increase Undernames</h1>
+        <h1 className="flex white text-2xl">Increase Undernames</h1>
```
Comment (kunstmusik) [inline src/components/layout/UpgradeUndernames/UpgradeUndernames.tsx:116]: Modify to text-[2rem]

File: package.json (hunk 4 above)
Comment (atticusofsparta) [reply package.json:119]: no - will remove. After effect from the yarn berry thing.

**Comment by dtfiedler** [comment]
The UI looks good, however I had some issues.

When I first logged in `Upgrade Ants` said I would update 16 ANTS. I clicked `Approve` and `Updated`. The loading indicator showed the progress, and showed the error for one ANT that failed. Fair enough. When I returned to `Manage` Page, I clicked the`Upgrade ANTs` again expecting it to show `1 ANT needs to be updated` instead it still showed `16 ANTs need to be updated`. I went ahead anyway and saw the requests for 16 ANTs go through again, not just the 1.

image

So questions:
- how are you tracking which needs to be upgraded? Can we show that on  a column in the table (bases on that criteria)
- how can i individually update one ANT? - is that just another button on the row that says upgrade?

**Review by kunstmusik — APPROVED** [review]

**Pull request merged**